src/components/Property/PropertyDetailsGrid.js

```diff
@@ -22,12 +23,12 @@ const PropertyDetailsGrid = ({
             subTypeDescription={subTypeDescription}
             hasLinkToProperty={hasLinkToProperty}
           />
-
           <TenureAlertDetails
```

Comment:
Do you think this component might be better called just `TenureDetails` now? I wasn't sure if a TMO is technically an "alert" or just some additional info that might be useful to the viewer in certain circumstances